### PR TITLE
 [NCL-3392][NCL-3362] Adapt to new close milestone procedure 

### PR DESCRIPTION
**Pull request by thescouser89**
Example on how to invoke it:

```
➜  pnc-cli git:(ncl-3362) ✗ pnc close-milestone -w 18
Command performed with user: dcheung
Latest release for milestone is in progress, waiting till it finishes...
Status of release for milestone: SUCCEEDED
```

### CODE REVIEW

**Comment by thescouser89** [comment]
This PR also includes swagger api regeneration for the new endpoint.

File: pnc_cli/swagger_client/apis/bpm_api.py [hunk 1]
```diff
@@ -430,7 +430,7 @@ def start_r_creation_task_with_single_url(self, body, **kwargs):
                                             body=body_params,
                                             post_params=form_params,
                                             files=files,
-                                            response_type=int,
+                                            response_type=None,
```
Comment (janinko) [inline pnc_cli/swagger_client/apis/bpm_api.py]: this should remain int

Comment (thescouser89) [reply pnc_cli/swagger_client/apis/bpm_api.py]: I actually did check the backend code on that change (and also the moving funcs and fields around).

For the moving thing, I guess it's ok cause subsequent swagger api regeneration won't do those changes again? I dunno.

I'll double check if the response type should stay int or None though

Comment (janinko) [reply pnc_cli/swagger_client/apis/bpm_api.py]: I'm not sure it won't change when regenerating again. I think I tried that once before but second regeneration ended up differently (but can't be sure if the generator version was the same in both cases or if the api on PNC changed)

**Comment by janinko** [comment]
Why are you removing the Repository Creation api form the init files?

**Pull request merged**